Nodejs/Product/Nodejs/SharedProject/CommonPropertyPage.cs

```diff
@@ -347,9 +32,9 @@ void IPropertyPage.SetObjects(uint count, object[] punk)
             {
                 if (punk[0] is ProjectConfig)
                 {
-                    if (this._project == null)
+                    if (this.Project == null)
                     {
-                        this._project = (CommonProjectNode)((CommonProjectConfig)punk.First()).ProjectMgr;
+                        this.Project = (CommonProjectNode)((CommonProjectConfig)punk.First()).ProjectMgr;
                     }
 
                     var configs = new List<CommonProjectConfig>();
```

Comment:
Why are we building this list?

Reply:
If there's no need to do so, it seems like the null check can be lifted out a level.

Reply:
yep